app/models/decision_review.rb

```diff
@@ -103,6 +103,7 @@ def mark_rating_request_issues_to_reassociate!
   end
 
   def serialized_legacy_issues
+    return [] if !legacy_opt_in_approved
```

Comment:
this isn't correct. It should show the issues modal regardless if the opt in was approved or not. It should not show them if the feature flag was turned off.

```suggestion
    return [] if !FeatureToggle.enabled?(:intake_legacy_opt_in)
```

Reply:
`legacy_opt_in_approved` will toggle whether the matched issue is eligible or not.

Reply:
I forgot that we still need the legacy issues for ineligibility